File: RadioControl/app/src/main/java/com/nikhilparanjape/radiocontrol/holders/RecyclerHolder.java
package com.nikhilparanjape.radiocontrol.holders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.nikhilparanjape.radiocontrol.R;

public class RecyclerHolder extends RecyclerView.ViewHolder {

    public TextView nametxt;

    public RecyclerHolder(View itemView) {
        super(itemView);

        nametxt= itemView.findViewById(R.id.nameTxt);
    }
}

File: RadioControl/app/src/main/java/com/nikhilparanjape/radiocontrol/holders/RecyclerHolder.kt
package com.nikhilparanjape.radiocontrol.holders

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.nikhilparanjape.radiocontrol.R


/**
 * Created by Nikhil on 10/12/2018.
 *
 * An holder class for TroubleShooting Activity
 */
class RecyclerHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var nametxt: TextView = itemView.findViewById(R.id.nameTxt)

}